tests/ttnn/integration_tests/resnet/test_ttnn_functional_resnet50_new.py

```diff
@@ -197,7 +197,7 @@ def __init__(self, device, batch_size, act_dtype, weight_dtype, math_fidelity):
         self.weight_dtype = weight_dtype
         self.math_fidelity = math_fidelity
 
-        torch_model = torchvision.models.resnet50(weights=torchvision.models.ResNet50_Weights.IMAGENET1K_V1).eval()
```

Comment:
could you also remove the `skip_for_wormhole` from line 273 to enable RN50 on WH (b=16). b=20 is skipped in the function.